dopex-io/elvarg
# feat: metadata seo (DOP-383)

## Scope



[closes issue-DOP-383](hhttps://linear.app/dopex/issue/DOP-383/add-dopex-metadata-portfolio-page)

## Implementation

Used NextSeo and the instructions provided on linear

## Screenshots

Not available

## How to Test

It should be correct to insert the full url (https//dopex.io/*) but this doesn't allow us to test before we merge
Then we can use open graph debugging tool
